apps/the_monkeys/src/app/settings/components/security/Danger.tsx

```diff
@@ -30,8 +30,8 @@ export const Danger = ({ data }: { data?: IUser }) => {
         title: 'Success',
         description: 'Your account has been deleted successfully.',
       });
-
-      router.replace('/');
+      queryClient.setQueryData(['auth'], null);
+      router.replace('/feed'); //optional: fallback to redirect user to feed
```

Comment:
/feed will be for logged in users hence redirect to the host URL(monkeys.com.co)